pantheon.yml

```diff
@@ -1,7 +1,16 @@
 api_version: 1
 web_docroot: true
+php_version: 7.4
+
+# See https://pantheon.io/docs/pantheon-yml#specify-a-version-of-mariadb
+database:
+  version: 10.4
+
+# See https://pantheon.io/docs/pantheon-yml/#enforce-https--hsts for valid values.
 enforce_https: transitional
-php_version: 7.3
-drush_version: 10
```

Comment:
Is it expected to remove this?

Reply:
Not necessary for Composer-managed sites. Removed for consistency with other pantheon.upstream.yml files in other standard upstreams.